tools/mod/go.mod

```diff
@@ -7,70 +7,76 @@ require (
 	github.com/chzchzchz/goword v0.0.0-20170907005317-a9744cb52b03
 	github.com/coreos/license-bill-of-materials v0.0.0-20190913234955-13baff47494e
 	github.com/gogo/protobuf v1.3.2
-	github.com/google/addlicense v0.0.0-20210428195630-6d92264d7170
-	github.com/gordonklaus/ineffassign v0.0.0-20200809085317-e36bfde3bb78
-	github.com/grpc-ecosystem/grpc-gateway v1.14.6
+	github.com/google/addlicense v1.0.0
+	github.com/gordonklaus/ineffassign v0.0.0-20210914165742-4cc7213b9bc8
+	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/gyuho/gocovmerge v0.0.0-20171205171859-50c7e6afd535
-	github.com/hexfusion/schwag v0.0.0-20170606222847-b7d0fc9aadaa
+	github.com/hexfusion/schwag v0.0.0-20211117114134-3ceb0191ccbf
 	github.com/mdempsky/unconvert v0.0.0-20200228143138-95ecdbfc0b5f
-	github.com/mgechev/revive v1.0.2
-	github.com/mikefarah/yq/v3 v3.0.0-20201125113350-f42728eef735
+	github.com/mgechev/revive v1.2.0
+	github.com/mikefarah/yq/v3 v3.0.0-20201202084205-8846255d1c37
```

Comment:
probably we can upgrade to `github.com/mikefarah/yq/v4` directly?

Reply:
Good catch. Done!